packages/bitcore-client/bin/wallet-send

```diff
@@ -40,14 +43,21 @@ const main = async () => {
     let feeRate = feeRateParam;
     if (!feeRate) {
       const feerate = await wallet.getNetworkFee();
-      feeRate = JSON.parse(feerate).feerate;
+      feeRate = feerate.feerate;
     }
     let nonce;
-    if (!['BTC', 'BCH', 'DOGE', 'LTC'].includes(chain)) {
+    let feeUnit;
+    if (wallet.chain.toUpperCase() === 'XRP') {
+      feeRate = feeRateParam == null ? feeRate : xrpl.xrpToDrops(feeRateParam); // convert feeRateParam from XRP to drops
```

Comment:
Add this here so that the sequence (XRP name for nonce) is updated correctly:
`nonce = await wallet.getNonce();`